apps/freelance/src/pages/PasswordReset/index.tsx

```diff
@@ -85,9 +83,25 @@ const PasswordReset = () => {
           />
         </Form.Item>
         <Form.Item>
-          <Button type="primary" htmlType="submit" loading={loading}>
+          <Button type="primary" htmlType="submit" loading={isLoading}>
             {t('resetPassword.buttonText')}
           </Button>
+          {isError && (
+            <StyledError>
+              {t(
+                errorMessages.find(
+                  obj =>
+                    obj.error ===
+                    JSON.parse(JSON.stringify(error))?.data?.message,
+                )?.message || 'resetPassword.errors.unexpected',
```

Comment:
its look weard, if all this transformation needed, create some function with name that will describe what you actually do there

Reply:
@lyudad fixed